Merge package template and update it (#1)
toitware/toit-ibutton: README.md

# iButton

A library to interact with iButtons, such as the DS1990A.

> This library also supports the RW1990 which allows to modify the ID of the iButton.

Reviewer: ```suggestion
This library also supports the RW1990 which can modify the ID of the iButton.
```
